packages/hint-manifest-icons/tsconfig.json

```diff
@@ -11,5 +11,9 @@
     "include": [
         "src/**/*.ts",
         "tests/**/*.ts"
+    ],
+    "references": [
+        { "path": "../hint" },
+        { "path": "../utils-tests-helpers" }
```

Comment:
Guess we missed this in the PR for `hint-manifest-icons`? Probably should keep this as a separate commit unless it doesn't matter since we haven't released it yet.

Reply:
I thought I had pushed those changes yesterday to master :/